source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
+      :ref:`Atlas admin <atlas-user-privileges>` role. The user must
+      also be able to :ref:`read the oplog <ref-atlas-oplog>`.
+
+      Destination Cluster Authentication Requirements
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      If the destination cluster is hosted in Atlas, the user must have
+      the :ref:`Atlas admin <atlas-user-privileges>` role.
+
+      Cluster Authentication Notes
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      - To add an Atlas user, see: :ref:`mongodb-users`.
+      - To add a user to a self-managed cluster, see: :ref:`create-users`.
+      - To verify user permissions, run :method:`db.getUser()`.
+
+   .. step:: Download and install ``mongosync``
+
+      :ref:`c2c-mongosync` is the tool that connects the source and
+      destination clusters. You can host ``mongosync`` on its own
+      hardware, ``mongosync`` does not have to run on the hardware that
+      hosts your MongodDB clusters.
+
+      To install ``mongosync``:
+      
+      #. Locate a host server for the ``mongosync`` executable that has
+         network connectivity to your source and destination clusters.
+      #. Go to the {+mdb-download-center+}.
+      #. Download the ``mongosync`` package for your host system.
+      #. Unpack the ``mongosync`` package. The ``mongosync`` executable
+         is in the ``bin`` directory. 
+
+      For operating system specific installation instructions, see
+      :ref:`c2c-install`.
+
+   .. step:: Connect the Clusters
+
+      A connection string contains the network and authentication
+      details that ``mongosync`` needs to connect to the source and
+      destination clusters.
+
+      Determine the IP address and port for your source and destination
+      clusters. You will use this information and the user
+      authentication details to construct the connection strings. 
+
+      The standard URI connection scheme has the form:
+
+      .. code-block:: shell
+
+         mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]
+
+      Your connections strings will resemble:
+
+      .. code-block:: shell
+
+         cluster0:
+         mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020
+         cluster1:
+         mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020
+
+      For more details, see :ref:`c2c-connecting`.
+
+Initialize ``monogsync``
+------------------------
+
+``monogsync`` must create an initial connection to the source and
+destination clusters before it can start to sync data. To create the
+initial connection, issue the following command on a single line (the
+command is reformated here for clarity):
+
+.. code-block:: shell
+
+   ./bin/mongosync --cluster0 mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,
```

Comment:
Making this multi-line means we can't copy/paste this into a terminal. Either make it one line, or include `\` line breaks where appropriate.